### PR TITLE
Core/Access: fix db and core missmatch and some small fixes to LFG item level cheks

### LINKED ISSUE #1934: item_level under table access_requirement does not work
item_level under table access_requirement does not work

##### SMALL DESCRIPTION:

item_level under table access_requirement does not work

##### EXPECTED BLIZZLIKE BEHAVIOUR:

when setting an item level, if the player does not have that item level he cannot enter instance

##### CURRENT BEHAVIOUR:

item_level under table access_requirement does not work, players can enter naked with no gear in bags etc

##### STEPS TO REPRODUCE THE PROBLEM:

1.  Set item level of instance (ulduar for example) to 200
2.  Enter raid on naked character
3.  No message displays, characters can enter naked with no issues

##### BRANCH(ES):

master

##### AC HASH/COMMIT:

6480f99ca2f4+

##### OPERATING SYSTEM:

UBUNTU LINUX 16.04

##### OTHER CUSTOMIZATIONS:

<bountysource-plugin>

---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/74977251-item_level-under-table-access_requirement-does-not-work?utm_campaign=plugin&utm_content=tracker%2F40032087&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F40032087&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by Viste**
##### CHANGES PROPOSED:
-  in title
##### HOW TO TEST THE CHANGES: your problem
in general need check lfg
##### Target branch(es):Master

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,4 +64,4 @@ This project is brought to you thanks to:
 
 List of organizations that help the AzerothCore project:
 
-[![JetBrains](https://user-images.githubusercontent.com/75517/51205146-7f225c80-1905-11e9-82e0-835627be170d.png)](https://www.jetbrains.com/?from=AzerothCore)
+[![JetBrains](https://user-images.githubusercontent.com/75517/51205146-7f225c80-1905-11e9-82e0-835627be170d.png)](https://www.jetbrains.com/?from=AzerothCore)
```
Comment (FrancescoBorzi) [inline README.md]: ?

Comment (Viste) [reply README.md]: don't know whats up maybe git get insane ?

Comment (BarbzYHOOL) [reply README.md]: invisible character from MVS I guess, i don't remember the name, "bom mark" or smthg like that

**Comment by BarbzYHOOL** [comment]
@disclosurez test this!!!!!!!!!!!!!!

File: README.md (hunk 1 above)
Comment (Viste) [reply README.md]: MVS on linux?  NICE!

**Comment by Viste** [comment]
btw second part of PR fixes recheck GS on rejoin to Dungeon Find in some cases

**Comment by FrancescoBorzi** [comment]
@Viste why closing this?

**Comment by Viste** [comment]
need split to 2 seprate

**Pull request closed without merging**